fenicsprecice/adapter_core.py

```diff
@@ -218,10 +218,7 @@ def get_fenics_vertices(function_space, coupling_subdomain, dims):
         if coupling_subdomain.inside(v.point(), True):
             lids.append(v.index())
             gids.append(v.global_index())
-            if dims == 2:
-                coords.append([v.x(0), v.x(1)])
-            if dims == 3:
-                coords.append([v.x(0), v.x(1), v.x(2)])
+            coords.append([v.x(d) for d in range(dims)])
```

Comment:
Please add an assertion that `dims == 2 or dims == 3`. Ideally at the beginning of the function.

Reply:
I do not see the need to add an assertion here. This is an internal function of the adapter and is called in `initialize()` where we already check the dimensions. There is no possibility of an arbitrary dimension being passed here.

Reply:
Then let's merge this PR right away ;)